addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6818,7 +6818,18 @@ msgctxt "#13516"
 msgid "Add art"
 msgstr ""
 
-#empty strings from id 13517 to 13549
+#: system/settings/settings.xml
+msgctxt "#13517"
+msgid "Threshold when atempo resampling kicks in"
```

Comment:
Then let my ignorance be an example of one of those confused users I mentioned :) The takeaway is that this needs to be explained much, much better.